modules/installing-ocp-agent-boot.adoc

```diff
@@ -34,9 +34,9 @@ This is the preferred method for the Agent-based installation. Using {ztp} manif
 
 . Create the `install-config.yaml` file:
 +
-[source,yaml]
+[source,terminal]
 ----
-cat << EOF > ./my-cluster/install-config.yaml
+$ cat << EOF > ./my-cluster/install-config.yaml
```

Comment:
@bergerhoffer decided to implement your suggestions for this file as well, while I was at it.